dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -36,7 +36,19 @@ cmd_config() {
     docker_run -v "${CHE_HOST_CONFIG}":/copy \
                -v "${CHE_HOST_DEVELOPMENT_REPO}"/dockerfiles/init:/files \
                   $IMAGE_INIT
+  fi
+
+  info "config" "Generating $CHE_MINI_PRODUCT_NAME configuration..."
+  # Run the docker configurator
+  generate_configuration_with_puppet
+
+  # Replace certain environment file lines with their container counterparts
+  info "config" "Customizing docker-compose for running in a container"
+
+  # Write the installed version to the *.ver file into the instance folder
+  echo "$CHE_VERSION" > "${CHE_CONTAINER_INSTANCE}/${CHE_VERSION_FILE}"
```

Comment:
This line should not be here. The only time we should ever write the version file is init(), or as part of a re-init, which is part of an upgrade.

The new CLI is structured so that once you have init() a folder, you cannot change the version using che.env. We removed that ability as it introduced user errors.